test/suite/test_checkpoint_snapshot05.py

```diff
@@ -147,13 +147,13 @@ def test_checkpoint_snapshot(self):
         try:
             ckpt.start()
             
-            # Wait for checkpoint to start before committing.
-            ckpt_started = 0
-            while not ckpt_started:
+            # Wait for checkpoint to start and acquire its snapshot before committing.
+            ckpt_snapshot = 0
+            while not ckpt_snapshot:
+                time.sleep(1)
```

Comment:
Is this sleep necessary at all anymore? Maybe it could be 0.01 or something shorter than a second if so?

Reply:
I do think the sleep is still necessary as we want to wait until the checkpoint has started and acquired a snapshot before doing anything with the transaction. On my end, it doesn't seem like reducing the sleep to 0.01 does much, is this the case for you as well?

Reply:
Talked offline, will open a new ticket if needed later.